### PR TITLE
add migrations, setup instructions

### PR DESCRIPTION
**Pull request by mastfissh**
This makes it easier to develop on, by explaining how to get the doctests to pass.

### CODE REVIEW

File: src/schema.rs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+table! {
+    users (id) {
+        id -> Int4,
+        name -> Text,
+        age -> Int4,
+    }
+}
```
Comment (davidpdrsn) [inline src/schema.rs]: Do we need this for the tests? 🤔

Comment (mastfissh) [reply src/schema.rs]: Nope! Removed.

Guess that was one of the potentially unwanted things diesel setup does...

**Comment by mastfissh** [comment]
Closing and re-opening to try to get travis to test

File: migrations/2019-05-06-130337_create_users/down.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+-- This file should undo anything in `up.sql`
```
Comment (davidpdrsn) [inline migrations/2019-05-06-130337_create_users/down.sql]: I guess we need `DROP TABLE users;` here.

File: diesel.toml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# For documentation on how to configure this file,
+# see diesel.rs/guides/configuring-diesel-cli
+
+[print_schema]
+file = "src/schema.rs"
```
Comment (davidpdrsn) [inline diesel.toml]: This file can be deleted as well right?

**Pull request merged**